grammar/utils.js

```diff
@@ -35,6 +35,80 @@ class Utils {
         return Address.columnNameToNumber(columnName);
     }
 
+    parseAddress(address) {
+      const range = address.split(":")
+      const cells = range.map((address) => this.parseCellAddress(address))
+      if(cells.length == 1){
+        return cells[0]
+      }else if(cells.length == 2) {
+        return {
+          ref: { 
+            from: {
+              col: Math.min(cells[0].ref.col, cells[1].ref.col),
+              row: Math.min(cells[0].ref.row, cells[1].ref.row)
+            },
+            to: {
+              col: Math.max(cells[0].ref.col, cells[1].ref.col),
+              row: Math.max(cells[0].ref.row, cells[1].ref.row)
+            }
+          }
+        }
+      }else {
+        throw "Unreachable Code Error"
```

Comment:
This error message should include the value of cells so we can see what value of cells cause a unreachable code error